client/vfm-react-app/src/pages/Register.js

```diff
@@ -37,17 +45,52 @@ export const Register = () => {
     });
   };
 
+  React.useEffect(() => {
+   setCredentials({
+    user_id: Math.round(Math.random() * 10000) + 10,
```

Comment:
We should probably let the database create an idea and get that ID in the response from the server if successful, otherwise there is a change we will have conflicts, right?